src/frontend/app/shared/components/service-icon/service-icon.component.scss

```diff
@@ -0,0 +1,11 @@
+.service-icon {
+  align-self: flex-end;
+  display: flex;
+  padding-right: 24px;
```

Comment:
This right padding is only needed in the case where we have a top right gripper menu